src/java/picard/sam/ValidateSamFile.java

```diff
@@ -58,20 +58,29 @@
 )
 public class ValidateSamFile extends CommandLineProgram {
     static final String USAGE_SUMMARY = "Validates a SAM or BAM file.  ";
-    static final String USAGE_DETAILS = "This tool reports on the validity of a SAM or BAM file relative to the SAM format specification " +
-            "(see http://samtools.github.io/hts-specs/SAMv1.pdf), which is useful for troubleshooting errors encountered with other tools " +
-            "that may be caused by improper formatting.<br /><br />" +
-            "By default, the tool runs in VERBOSE mode and will exit after finding 100 errors and output them to the " +
-            "console (stdout). It is often practical to start by running this tool with the SUMMARY mode option, which summarizes the " +
-            "\"errors\" and \"warnings\". Consequently, specific validation warnings or errors that are of lesser concern can be ignored " +
-            "using the IGNORE and/or IGNORE_WARNINGS arguments in order to focus on blocking errors. " +
-            "<br />" +
+    static final String USAGE_DETAILS = "<p>This tool reports on the validity of a SAM or BAM file relative to the SAM format " +
+            "specification.  This is useful for troubleshooting errors encountered with other tools that may be caused by improper " +
+            "formatting, faulty alignments, incorrect flag values, etc. </p> " +
+
+            "<p>By default, the tool runs in VERBOSE mode and will exit after finding 100 errors and output them to the console (stdout). " +
+            "Therefore, it is often more practical to run this tool initially using the MODE=SUMMARY option.  This mode outputs a summary " +
+            "table listing the numbers of all 'errors' and 'warnings'.</p> "+
+
+            "<p>When fixing errors in your file, it is often useful to prioritize the severe validation errors and ignore the " +
+            "errors/warnings of lesser concern.  This can be done using the IGNORE and/or IGNORE_WARNINGS arguments.  For helpful " +
+            "suggestions on error prioritization, please follow this link to obtain additional documentation on <a href='http://gatkforums.broadinstitute.org/gatk/discussion/6956'>ValidateSamFile</a>.</p>" +
```

Comment:
This link is broken -- I think you mean https://www.broadinstitute.org/gatk/guide/article?id=7571